### PR TITLE
Update Cli.md

### PR DESCRIPTION
**Pull request by sanathusk**
Since the standalone pub has been deprecated ( refer [here](https://dart.dev/tools/pub/cmd ) added instructions on how to use the pub bundled with dart instead.

### CODE REVIEW

**Review by sfshaza2 — APPROVED** [review]
Thanks for catching that!

**Comment by sfshaza2** [comment]
@sanathusk, I can't merge this PR until you sign the CLA.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Approved, once CLA is signed.

**Review by 69Assassin — APPROVED** [review]
Sorry

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Comment by sfshaza2** [comment]
Closing.

**Comment by sanathusk** [comment]
@googlebot I signed it!

**Pull request closed without merging**
